site/source/user-manual/service-guides/lightning/lightning-integrations.rst

```diff
@@ -97,7 +97,7 @@ Available For
 .............
 - embassyOS
 
-See the video at the top of this page for an intro to Lightning using RTL.
+This can simply be installed by going to your Start9 server's marketplace clicking on "Ride The Lightning" then installing the latest version. Once installed you can configure it to work with either - or both - Core Lightning and LND!
```

Comment:
```suggestion
This can simply be installed by going to your Start9 Server's marketplace clicking on "Ride The Lightning" then installing the latest version. Once installed you can configure it to work with either - or both - Core Lightning and LND!
```